package.json

```diff
@@ -289,7 +289,7 @@
 		"@types/should": "^11.2.0",
 		"@types/sinon": "^17.0.4",
 		"@types/turndown": "^5.0.5",
-		"@types/vscode": "^1.84.0",
+		"@types/vscode": "=1.84.0",
```

Comment:
Is there a reason we are locking the types to 1.84.0 here?

Reply:
There's some part of the code that breaks with 1.100.0 in my testing. If I recalled correctly it is the file `/src/api/providers/vscode-lm.ts`.